### ISSUE TITLE
Accessibility improvements

### ISSUE THREAD

**Issue by dominikklein** [formkit/formkit#123]
### Describe the feature

I think there a some improvements needed for the general accessibility.

For example at the moment the `aria-describedby` is not used by default for the error messages.

**Comment by justin-schroeder** [comment]
Agree @dominikklein — more needs to be done here. I just pushed an update in beta.5 that adds `aria-describedby` to all of the input elements.

We'd like to get the entire thing professionally audited to make sure we have a really solid foundation for other to build on.

**Comment by dominikklein** [comment]
Yes i think it's a complex topic to make this work in the best way.
For example you added now the aria-describedby for the "help" text, but if a error exists it's maybe also this connection which should be present.

**Comment by justin-schroeder** [comment]
@dominikklein so this latest update should targets any help text, validation messages, or error messages with aria-describedby. Also the validation/error messages are now in a `aria-live="polite"` — which could debatably be assertive instead — but I did use assertive for the error messages placed on the form itself — like if you submit a form and it fails.

**Comment by justin-schroeder** [comment]
I think all the basics are covered now so I'm going to close this in favor of specific, addressable, accessibility issues that arise in the future.
